src/pages/Patients/Patients.js

```diff
@@ -153,6 +151,19 @@ const Patients = () => {
     [patients],
   );
 
+  const getPatientRiskData = async () => {
+    const patientRisk = await patientService.fetchPatientRiskData();
+    setPatientRisk(patientRisk.data.riskCount);
+    const totalPatients = patientRisk.data.riskCount.filter(
+      (patient) => patient.status === 'All',
+    );
+    const dischargedPatients = patientRisk.data.riskCount.filter(
+      (patient) => patient.status === 'Discharged',
+    );
+    setTotalPatients(totalPatients[0]?.count);
+    setDischargedPatients(dischargedPatients[0]?.count);
```

Comment:
```suggestion
    const totalPatients = patientRisk.data.riskCount.find(
      (patient) => patient.status === 'All',
    );
    const dischargedPatients = patientRisk.data.riskCount.find(
      (patient) => patient.status === 'Discharged',
    );
    setTotalPatients(totalPatients?.count);
    setDischargedPatients(dischargedPatients?.count);
```

Using `find` is a better way to do this than using `filter` because we know there will only be one element in the array with the desired value

Reply:
resolved